api/envoy/config/wasm/v2/wasm.proto

```diff
@@ -10,43 +10,60 @@ import "envoy/api/v2/core/base.proto";
 
 import "validate/validate.proto";
 
-// [#protodoc-title: Metrics service]
+// [#protodoc-title: Wasm service]
 
+// Configuration for a Wasm VM.
 message VmConfig {
+  // A unique ID for a VM permitting multiple filters/services can share the same VM.
+  // If left blank the 'vm_id' will be a hash of the Wasm code enabling the VM to persist over
+  // reconfigurations if the code does not change. This will also result in filters/services running
+  // the same Wasm code sharing the same VM. If set, 'vm_id' should include a version number
+  // otherwise the VM will not be reloaded if the Wasm code changes.
+  string vm_id = 1;
+
   // The Wasm VM type (see source/extensions/commmon/wasm/well_known_names.h).
-  string vm = 1;
+  string vm = 2;
```

Comment:
Since we're already changing protos, maybe rename this to `runtime` (or `engine`)?

Reply:
Done: runtime